charts/portworx/values.yaml

```diff
@@ -62,8 +62,41 @@ customRegistryURL:
 registrySecret:
 licenseSecret:
 
-monitoring: false
-telemetry: false
+monitoring:
+  prometheus:                         #
+    enabled: false                     # Enable prometheus
+    exportMetrics: true               # Expose the Portworx metrics to an external or operator deployed Prometheus.
+    alertManager: false               # Enable prometheus alert manager
+    resources:                        # Configure Prometheus resource usage such as memory and CPU usage
+      # requests:
+      #   memory: "64Mi"
+      #   cpu: "250m"
+      # limits:
+      #   memory: "128Mi"
+      #   cpu: "500m"
+    replicas: 0                       # Number of prometheus replicas that will be deployed. Default is 1
+    retention: ""                     # Time period for which prometheus retains historical matrics. default is 24h.
```

Comment:
let's put the default values for such things.

Reply:
similar to replica these defaults can be set by operator

Reply:
Done Added default value as 24h